docs/faq.md

```diff
@@ -33,7 +33,16 @@ library(SingleCellExperiment)
 scpca_sample <- readRDS("SCPCL000000_filtered.rds")
 ```
 
-## What is the difference between a sample and a library?
+## What is the difference between participants, samples, and libraries?
+
+The `participant_id` indicates the participant, or patient, from which a sample, or collection of samples, is obtained from. 
```

Comment:
We don't have participant ids for all samples right? Only when there is more than one sample from the same participant? If so, we should say that explicitly.  (Maybe in a separate FAQ entry)

I simplified here, mostly because there were lots of commas that I think were getting confusing.
```suggestion
The `participant_id` indicates the participant from which a sample or collection of samples was obtained. 
```